samples/nrf9160/nrf_cloud_mqtt_multi_service/prj.conf

```diff
@@ -0,0 +1,94 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+
+# TODO: What about for Thingy? Should there be a UI?
+# DK
+CONFIG_DK_LIBRARY=y
+
+# Log level
+# For more verbose and detailed log output, set the log level to
+# CONFIG_MQTT_MULTI_SERVICE_LOG_LEVEL_DBG=y instead.
+CONFIG_MQTT_MULTI_SERVICE_LOG_LEVEL_INF=y
+
+# General config
+CONFIG_EVENTS=y
+CONFIG_NEWLIB_LIBC=y
+CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
+CONFIG_RESET_ON_FATAL_ERROR=y
+CONFIG_NCS_SAMPLES_DEFAULTS=y
+
+# Heap and stacks
+# Extended memory heap size needed both for PGPS and for encoding JSON-based nRF Cloud Device Messages
+CONFIG_HEAP_MEM_POOL_SIZE=24576
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+# Network
+CONFIG_NETWORKING=y
+CONFIG_NET_NATIVE=n
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_OFFLOAD=y
+
+# Modem library
+CONFIG_NRF_MODEM_LIB=y
+
+# AT commands interface
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_AT_HOST_LIBRARY=y
+
+# nRF Cloud
+CONFIG_NRF_CLOUD_MQTT=y # This also enables FOTA by implicitly setting CONFIG_NRF_CLOUD_FOTA=y
+
+# MQTT
+CONFIG_MQTT_KEEPALIVE=120
+
+# MCUBOOT - Needed by FOTA
+CONFIG_BOOTLOADER_MCUBOOT=y
+CONFIG_IMG_MANAGER=y
+CONFIG_MCUBOOT_IMG_MANAGER=y
+
+# Location Services configuration
+CONFIG_LOCATION=y
+CONFIG_LOCATION_METHOD_GNSS=y
+CONFIG_LOCATION_METHOD_CELLULAR=y
+CONFIG_NRF_CLOUD_AGPS=y
+CONFIG_NRF_CLOUD_CELL_POS=y
+CONFIG_MULTICELL_LOCATION_SERVICE_NRF_CLOUD=y
+CONFIG_MODEM_INFO=y
+CONFIG_MODEM_INFO_ADD_NETWORK=y
+CONFIG_NRF_CLOUD_PGPS=y
+CONFIG_NRF_CLOUD_PGPS_REPLACEMENT_THRESHOLD=4
+CONFIG_NRF_CLOUD_PGPS_REQUEST_UPON_INIT=n
+
+# Date Time lib - Used by PGPS and main application
+CONFIG_DATE_TIME=y
+
+# LTE link control - used by PGPS and main application
+CONFIG_LTE_LINK_CONTROL=y
+CONFIG_LTE_NETWORK_MODE_LTE_M_GPS=y
+CONFIG_LTE_AUTO_INIT_AND_CONNECT=n
+
+# Settings - used by nRF Cloud library and PGPS
+CONFIG_SETTINGS=y
+CONFIG_SETTINGS_FCB=y
+CONFIG_FCB=y
+
+# Download Client - used by FOTA and (TODO: figure out: PGPS?)
```

Comment:
It's required by P-GPS.  Is there a specific concern?

From Kconfig.nrf_cloud_pgps:
```
menuconfig NRF_CLOUD_PGPS
	bool "Enable nRF Cloud Predicted GPS (P-GPS)"
	depends on MODEM_INFO
	depends on MODEM_INFO_ADD_NETWORK
	depends on DATE_TIME
	imply DOWNLOAD_CLIENT
	select STREAM_FLASH_ERASE
	select SETTINGS
	select CJSON_LIB

- [ ]

Reply:
No specific concern, just forgot to look into it. Thank you!